aiida/backends/sqlalchemy/migrations/versions/1de112340b18_django_parity_3.py

```diff
@@ -0,0 +1,222 @@
+depends_on = None
+
+# table name, column name, index name
+MISSING_STANDARD_INDEXES = (
+    ('db_dbauthinfo', ('aiidauser_id',), False, 'db_dbauthinfo_aiidauser_id_0684fdfb'),
+    ('db_dbauthinfo', ('dbcomputer_id',), False, 'db_dbauthinfo_dbcomputer_id_424f7ac4'),
+    ('db_dbcomment', ('dbnode_id',), False, 'db_dbcomment_dbnode_id_3b812b6b'),
+    ('db_dbcomment', ('user_id',), False, 'db_dbcomment_user_id_8ed5e360'),
+    ('db_dbgroup', ('user_id',), False, 'db_dbgroup_user_id_100f8a51'),
+    ('db_dblog', ('dbnode_id',), False, 'db_dblog_dbnode_id_da34b732'),
+    ('db_dbnode', ('ctime',), False, 'db_dbnode_ctime_71626ef5'),
+    ('db_dbnode', ('mtime',), False, 'db_dbnode_mtime_0554ea3d'),
+    ('db_dbnode', ('dbcomputer_id',), False, 'db_dbnode_dbcomputer_id_315372a3'),
+    ('db_dbnode', ('user_id',), False, 'db_dbnode_user_id_12e7aeaf'),
+)
+
+# table name, column name, index name
+MISSING_VARCHAR_INDEXES = (
+    ('db_dbcomputer', 'label', 'db_dbcomputer_label_bc480bab_like'),
+    ('db_dbgroup', 'label', 'db_dbgroup_name_66c75272_like'),
+    ('db_dbgroup', 'type_string', 'db_dbgroup_type_23b2a748_like'),
+    ('db_dblink', 'label', 'db_dblink_label_f1343cfb_like'),
+    ('db_dblink', 'type', 'db_dblink_type_229f212b_like'),
+    ('db_dblog', 'levelname', 'db_dblog_levelname_ad5dc346_like'),
+    ('db_dblog', 'loggername', 'db_dblog_loggername_00b5ba16_like'),
+    ('db_dbnode', 'label', 'db_dbnode_label_6469539e_like'),
+    ('db_dbnode', 'node_type', 'db_dbnode_type_a8ce9753_like'),
+    ('db_dbnode', 'process_type', 'db_dbnode_process_type_df7298d0_like'),
+    ('db_dbsetting', 'key', 'db_dbsetting_key_1b84beb4_like'),
+    ('db_dbuser', 'email', 'db_dbuser_email_30150b7e_like'),
+)
+
+# table name, column names, constraint name
+DROP_UNIQUE_CONSTRAINTS = (
+    ('db_dbauthinfo', ('aiidauser_id', 'dbcomputer_id'), 'db_dbauthinfo_aiidauser_id_dbcomputer_id_key'),
+    ('db_dbcomment', ('uuid',), 'db_dbcomment_uuid_key'),
+    ('db_dbcomputer', ('label',), 'db_dbcomputer_label_key'),
+    ('db_dbcomputer', ('uuid',), 'db_dbcomputer_uuid_key'),
+    ('db_dbgroup', ('label', 'type_string'), 'db_dbgroup_label_type_string_key'),
+    ('db_dbgroup_dbnodes', ('dbgroup_id', 'dbnode_id'), 'db_dbgroup_dbnodes_dbgroup_id_dbnode_id_key'),
+    ('db_dbgroup', ('uuid',), 'db_dbgroup_uuid_key'),
+    ('db_dblog', ('uuid',), 'db_dblog_uuid_key'),
+    ('db_dbnode', ('uuid',), 'db_dbnode_uuid_key'),
+    ('db_dbsetting', ('key',), 'db_dbsetting_key_key'),
+)
+
+# table name, column names, constraint name
+ADD_UNIQUE_CONSTRAINTS = (
+    ('db_dbauthinfo', ('aiidauser_id', 'dbcomputer_id'), 'db_dbauthinfo_aiidauser_id_dbcomputer_id_777cdaa8_uniq'),
+    ('db_dbcomment', ('uuid',), 'db_dbcomment_uuid_49bac08c_uniq'),
+    ('db_dbcomputer', ('label',), 'db_dbcomputer_label_bc480bab_uniq'),
+    ('db_dbcomputer', ('uuid',), 'db_dbcomputer_uuid_f35defa6_uniq'),
+    ('db_dbgroup', ('label', 'type_string'), 'db_dbgroup_name_type_12656f33_uniq'),
```

Comment:
Maybe this name change is also missing in Django 48 migration?
```suggestion
    ('db_dbgroup', ('label', 'type_string'), 'db_dbgroup_label_type_12656f33_uniq'),
```

Reply:
See above